src/librustc/middle/ty/util.rs

```diff
@@ -578,6 +578,16 @@ impl<'tcx> ty::ctxt<'tcx> {
         });
         let generics = adt.type_scheme(self).generics;
 
+        // RFC 1238: if the destructor method is tagged with the
```

Comment:
This whole method should be `!self.has_attr(dtor_method, "unsafe_destructor_blind_to_params")`.